test/e2e/e2e_test.go

```diff
@@ -34,6 +34,7 @@ var _ = Describe("E2E tests", func() {
 			err := e2e.RunAddonsTests()
 			if err != nil {
 				log.Println("error while running package test, deleting cluster", err)
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
@aman556 if `err != nil`, `Expect(err).NotTo(HaveOccurred())` will always fail right? As error would have occurred and that's why it would be inside the `if` block